src/pages/staff/staff.ts

```diff
@@ -5,11 +5,12 @@ import gql from 'graphql-tag';
 
 // We use the gql tag to parse our query string into a query document
 const MemberList = gql`
-  query { readMembers {
-    Email: Email,
-    Name: FirstName,
-    Surname: Surname
-}}`;
+  query { readMembers { edges { node {
+    Email
```

Comment:
You removed the mapping, understandable, but I believe leaving it in would be a good addition to demo the power of GraphQL?